Sources/Talkshoplive/Core/Chat/ChatProvider.swift

```diff
@@ -77,4 +77,23 @@ public class ChatProvider {
         print("Initialized Pubnub", pubnub!)
     }
     
+    
+    private func subscribeChannels(showId: String) {
+        Networking.getCurrentEvent(showId: showId, completion: { result in
```

Comment:
@mayuri-talkshoplive Can we call Show.getCurrentEvent instead of making an API request again here?

Reply:
For that, We wil need to store the currentEvent data to one instance and than we can use it directly. I will check and make changes accordingly.